Project/Fall2020_CSC403_Project/FrmLevel.cs

```diff
@@ -47,9 +49,17 @@ public partial class FrmLevel : Form {
 
     private void OnEnemyDeath(Enemy enemy)
     {
-      // Render enemy off of the screen
-      enemy.Icon.Visible = false;
-
+      // Render enemy off of the screen after death animation
+      enemy.Icon.LoadAsync("C:\\Users\\jbt01\\Desktop\\LaTech fall 2020\\csc403project\\Fall2020_CSC403_Project\\Project\\Fall2020_CSC403_Project\\data\\chardeath.gif");
```

Comment:
This will only work on your local machine. Consider adding the gif file to the project, and accessing the project files using the application directory with "System.AppDomain.CurrentDomain.BaseDirectory," or "System.Environment.CurrentDirectory."

Reply:
I've added the gif as a relative path and copied it to the Debug/data folder as well so it will run in the ide and from the exe. That is the method used for the other LoadAsync uses since it requires that path as a string.